NETProvider/src/FirebirdSql.Data.UnitTests/FbCommandTests.cs

```diff
@@ -730,6 +756,15 @@ public void NoCommandPlanTest()
 				Assert.DoesNotThrow(() => { plan=cmd.CommandPlan; });
 				Assert.IsEmpty(plan);
 			}
+			}
+			finally
+			{
+				using (var cmd = Connection.CreateCommand())
+				{
+					cmd.CommandText = "drop table NoCommandPlanTest";
```

Comment:
For this particular case. 
Do ```cmd.CommandPlan``` property need ```cmd.ExecuteNonQuery();``` to be called?
I've replaced it with Prepare and test passes and drop table not needed.